app/models/container_node.rb

```diff
@@ -72,17 +73,36 @@ def perf_rollup_parents(interval_name = nil)
     [ext_management_system] unless interval_name == 'realtime'
   end
 
-  def ipaddress
+  def kubernetes_hostname
     labels.find_by(:name => "kubernetes.io/hostname").try(:value)
   end
 
   def cockpit_url
-    URI::HTTP.build(:host => ipaddress, :port => 9090)
+    URI::HTTP.build(:host => kubernetes_hostname, :port => 9090)
   end
 
   def evaluate_alert(_alert_id, _event)
     # currently only EmsEvents from hawkular are tested for node alerts,
     # and these should automaticaly be translated to alerts.
     true
   end
+
+  supports :external_logging_support do
+    unless ext_management_system.respond_to?(:external_logging_route_name)
+      unsupported_reason_add(:external_logging_support, _('This provider type does not support Extrenal Logging'))
```

Comment:
Does it make sense to make this a dynamic supports method?  Why not just `support_not :external_logging`.  @durandom Thoughts here?